src/dotty/tools/dotc/repl/CombinedClassLoader.scala

```diff
@@ -0,0 +1,37 @@
+package dotty.tools
+package dotc
+package repl
+
+import scala.util.Try
+
+class CombinedClassLoader(loaders: List[ClassLoader]) extends ClassLoader {
```

Comment:
Why doesn't it work for us? Can't we replace `parentClassLoader` in `val classLoader` with the classloader given by sbt?